docs/src/domains/troubleshoot.md

```diff
@@ -2,36 +2,36 @@
 title: "Going Live - Troubleshooting"
 weight: 4
 description: |
-  If all steps above have been followed and the site still doesn't resolve (after waiting for the DNS update to propagate), here are a few simple self-help steps to take before contacting support.
+  If your site still doesn't resolve after the DNS changes, check the troubleshooting before contacting support.
 sidebarTitle: "Troubleshooting"
 ---
 
 {{% description %}}
 
 ## Verify DNS
 
-On the command line with OS X or Linux (or using the Linux subsystem for Windows) type `host www.example.com`:
+On the command line using macOS, Linux or the Linux subsystem for Windows, type:
+`host www.{{< variable "YOUR_DOMAIN" >}}`.
 
-The response should be something like:
+The response where `www.{{< variable "YOUR_DOMAIN" >}}` is `example.com` should be something like:
 
 ```text
-www.example.com is an alias for main-t2xxqeifuhpzg.eu.platform.sh.
-main-t2xxqeifuhpzg.eu.platform.sh has address 54.76.136.188
+www.{{<variable "YOUR_DOMAIN" >}} is an alias for main-def456-abc123.eu-2.platformsh.site.
+main-def456-abc123.eu-2.platformsh.site has address 192.0.2.1
```

Comment:
:clipboard: The whole point of having `example.com` above is to use it in the example
```suggestion
www.example.com is an alias for main-abcd123.abcdefgh1234567.eu.platformsh.site.
main-abcd123.abcdefgh1234567.eu.platformsh.site has address 192.0.2.1
```